src/screens/CreatePost/index.js

```diff
@@ -249,39 +247,13 @@ const CreatePost = () => {
     }
   ];
 
-  const fetchMyProfile = async () => {
-    setLoading(true);
-    const userId = await getUserId();
-    if (userId) {
-      const result = await getMyProfile(userId);
-      if (result.code === 200) {
-        setDataProfile(result.data);
-        setLoading(false);
-        handleLocation(result.data);
-      }
-
-      setLoading(false);
-    }
-  };
-
-  React.useEffect(() => {
-    getSpecificCache(PROFILE_CACHE, async (res) => {
-      if (!res) {
-        fetchMyProfile();
-      } else {
-        setDataProfile(res);
-        handleLocation(res);
-      }
-    });
-  }, []);
-
   const handleLocation = async (res) => {
     setGeoList([...location, ...res?.locations]);
     setLoading(false);
   };
 
   React.useEffect(() => {
-    fetchMyProfile();
+    handleLocation();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to simplify the fetching of profile data and handling of location data. However, it's important to ensure that the `handleLocation` function is correctly defined and that it's being called with the right arguments. Also, make sure that the removal of `fetchMyProfile` doesn't affect other parts of the code.
```diff
-  const fetchMyProfile = async () => {
-    setLoading(true);
-    const userId = await getUserId();
-    if (userId) {
-      const result = await getMyProfile(userId);
-      if (result.code === 200) {
-        setDataProfile(result.data);
-        setLoading(false);
-        handleLocation(result.data);
-      }
-
-      setLoading(false);
-    }
-  };
-
-  React.useEffect(() => {
-    getSpecificCache(PROFILE_CACHE, async (res) => {
-      if (!res) {
-        fetchMyProfile();
-      } else {
-        setDataProfile(res);
-        handleLocation(res);
-      }
-    });
-  }, []);
-
-   React.useEffect(() => {
-    fetchMyProfile();
-   }, []);
+  React.useEffect(() => {
+   handleLocation();
+ }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->